pkg/util/errors.go

```diff
@@ -0,0 +1,60 @@
+package util
+
+import "errors"
+
+const (
+	WrapErrorGetBARFailed = "get bucketAccessRequest failed"
```

Comment:
Why a separate message for wrapped errors?

Reply:
I moved the error wrap messages here so we could easy to reference constants, which could be used in testing. If we just left them as strings across the project, it would be more difficult to track and update error messages. I am not that committed to this, and I could remove this change if you think it's a good idea